hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -871,6 +871,19 @@ private boolean isKeyEmpty(OmKeyInfo keyInfo) {
      startKeyName, keyPrefix, maxKeys);
   }
 
+  @Override
+  public boolean recoverTrash(String volumeName, String bucketName,
```

Comment:
We don't need this any more, after adding OMTrashRecoverRequest.

Reply:
Fixed.

Reply:
Updated.